src/common/sorted-table/index.js

```diff
@@ -307,6 +333,12 @@ export default class SortedTable extends Component {
     })
   }, 500)
 
+  _executeGroupedAction = createSelector(
```

Comment:
That's right for a memoized selector, but a selector is simply a function which extract data from its params. It should not have any side effects.